README.md

```diff
@@ -45,31 +45,16 @@ The `RustFunction` construct creates a Lambda function with automatic bundling a
     $ rustup target add aarch64-unknown-linux-gnu
     ```
 
-3)  Install [Zig], using the instructions in their [installation guide] or via one of the options below.
-
-    1. Using [pip] (Python 3 required):
-
-        ```shell
-        $ pip install ziglang
-        ```
-
-    2. Using [npm]:
-
-        ```shell
-        $ npm install -g @ziglang/cli
-        ```
-
-4)  Use [`cargo`] to install _messense/cargo-zigbuild_:
+4)  Use [`cargo`] to install _cargo-lambda_:
```

Comment:
a minor thing, but you could also have it like *calavera/cargo-lambda* here if you wanted to; since the previous format was using *user/repo* essentially.